src/components/footer.tsx

```diff
@@ -38,21 +35,21 @@ const IconKnowMore = (props: KnowMoreProps) => {
 const Footer = (): JSX.Element => {
   return (
     <>
-      <footer className="border border-t-[1px] border-b-0 border-r-0 border-l-0 border-textcolors-boundary bg-darkshades-passive py-2 font-Plex">
-        <section className="grid sm:grid-cols-1 md:grid-cols-6 md:gap-2 lg:grid-cols-10">
-          <section className=" hidden items-center justify-start md:col-span-2 md:col-start-1 md:flex lg:col-span-3 lg:col-start-1 lg:flex">
+      <footer className="border border-t-[2px] border-b-0 border-r-0 border-l-0 border-textcolors-boundary bg-darkshades-passive py-2 font-Plex content-center">
+        <section className="p-7 flex flex-row">
```

Comment:
Just try centring the element. 
```suggestion
        <section className="p-7 flex flex-row items-center">
```

Reply:
Yes done.